easybuild/easyconfigs/t/TurboVNC/TurboVNC-2.2.6-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,69 @@
+easyblock = 'CMakeMake'
+
+name = 'TurboVNC'
+version = '2.2.6'
+
+homepage = 'https://www.turbovnc.org'
+description = """TurboVNC is a derivative of VNC (Virtual Network Computing) that is tuned to provide
+ peak performance for 3D and video workloads."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['TurboVNC-2.2.3_fix-libX11-libs.patch']
+checksums = [
+    '71ff553dbad24f8339e168fe90e09bc2b13c21e453f28276d0e5c9c408bd9de2',  # turbovnc-2.2.6.tar.gz
+    'c3e29c2a11e0cf8e37080b1eae6f35811f64d3c05dc1b58485cb17918c465dd5',  # TurboVNC-2.2.3_fix-libX11-libs.patch
+]
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.21.1'),
+]
+
+dependencies = [
+    ('X11', '20210802'),
+    ('pixman', '0.40.0'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('Mesa', '21.1.7'),
+    ('libGLU', '9.0.2'),
+    ('libglvnd', '1.3.3'),
+    ('OpenSSL', '1.1', '', True),
+]
+
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev'),
+]
+
+configopts = "-DTVNC_BUILDJAVA=0 -DTVNC_SYSTEMX11=1 "
+configopts += "-DX11_X11_LIB=$EBROOTX11/lib/libX11.%s " % SHLIB_EXT
+configopts += "-DX11_Xau_LIB=$EBROOTX11/lib/libXau.%s " % SHLIB_EXT
+configopts += "-DX11_SM_LIB=$EBROOTX11/lib/libSM.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libXext.%s " % SHLIB_EXT
+configopts += "-DX11_ICE_LIB=$EBROOTX11/lib/libICE.%s " % SHLIB_EXT
+configopts += "-DX11_Pixman_LIB=$EBROOTPIXMAN/lib/libpixman-1.%s.0 " % SHLIB_EXT
+configopts += "-DOPENGL_glu_LIBRARY=$EBROOTLIBGLU/lib/libGLU.%s " % SHLIB_EXT
+configopts += "-DOPENGL_egl_LIBRARY=$EBROOTLIBGLVND/lib/libEGL.%s " % SHLIB_EXT
+configopts += "-DOPENGL_opengl_LIBRARY=$EBROOTLIBGLVND/lib/libOpenGL.%s " % SHLIB_EXT
+configopts += "-DOPENGL_glx_LIBRARY=$EBROOTLIBGLVND/lib/libGLX.%s " % SHLIB_EXT
```

Comment:
@boegel could you add a comment about the removal of the previous OpenGL config option?